frappe/email/receive.py

```diff
@@ -215,23 +215,17 @@ def check_imap_uidvalidity(self, folder):
 				Communication.communication_medium == "Email"
 			).where(Communication.email_account == self.settings.email_account).run()
 
-			if self.settings.use_imap:
```

Comment:
I have removed that block, as the whole method is dedicated to fetching mails via IMAP. I guess this has been missed, when refactoring the logic from v13 to v14 recently.